widgets/filters/src/FiltersTable.tsx

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+
+import type { Filter, Widget } from './types';
+
+interface FiltersTableData {
+    filters: Filter[];
+    total: number;
+}
+
+interface FiltersTableProps {
+    data: FiltersTableData;
+    toolbox: Stage.Types.Toolbox;
+    widget: Widget;
+}
+
+export default function FiltersTable({ data, toolbox, widget }: FiltersTableProps) {
```

Comment:
Optional:

I suggest using the `FunctionComponent` or `FC` type from `react` as a type annotation here:

```suggestion
const FiltersTable: FunctionComponent<FiltersTableProps> = ({ data, toolbox, widget }) => {
```

and then manually
```ts
export default FiltersTable;
```

The benefits are:
1. The return type is strictly validated (here you could have returned anything, no type safety for the return type)
1. The function is automatically detected as a component, thus allowing you to specify `displayName`, `defaultProps`, `propTypes` etc (does not apply here, just FYI)
1. `children` are automatically included in the props (does not apply here, though, just FYI)

Given that only one benefit applies here, treat this comment more as a general note for the future :slightly_smiling_face:

Reply:
If I do this, eslint forces me to add `propTypes`. My understanding is that I don't need it if this component is used only in TS code base, right?

Reply:
Hmm... right. I guess we should do something about this rule, since it may get more and more annoying. Maybe we could disable it today :smile: A topic for the weekly?

Reply:
> A topic for the weekly?

Sure

Reply:
I applied your suggestion and disabled `react/prop-types` rule for tsx files.